force-app/main/default/classes/searchAndPick.cls

```diff
@@ -0,0 +1,27 @@
+global with sharing class searchAndPick {
+          private final Account person;
+          public searchAndPick(ApexPages.StandardController stdController){
+    this.person= (Account)stdController.getRecord();
+  }
+    
+     public list <Contact> ctt {get;set;}
+    public list <Contact> cttt {get;set;}
+    public list <Contact> ctct {get;set;}
```

Comment:
Please seelct more understandable names and try to avoid defining same structure multiple times. try to reuse them